lte/gateway/deploy/roles/trfserver/files/traffic_server.py

```diff
@@ -559,9 +564,15 @@ def _run_iperf3(self, results_buffer, iperf):
         '''
         # Constructing the subprocess call
         params = ('-B', iperf.bind_address, '-p', str(iperf.port), '-J')
+        if ipaddress.ip_address(iperf.bind_address).version == 6:
+            params += ('-6',)
         if 'c' == iperf.role:
             params = ('-c', iperf.server_hostname) + params
             params += ('-b', str(iperf.bandwidth), '-t', str(iperf.duration))
+            # For ipv6 there is delay in configuring ipv6 address on eth3
+            # interface of test vm, so sleep for 5 secs
+            if ipaddress.ip_address(iperf.bind_address).version == 6:
+                time.sleep(5)
```

Comment:
Can we confirm that the interface is configured before moving forward?

Reply:
Since the UE IP address will be configured on test vm and this code is on trfserver we cannot know unless test vm sends a message. I thought adding sleep would be simpler than introducing new message

Reply:
I see! In that case, if the test fails at a later stage due to missing IP on gateway VM, is there a way to indicate that the cause of failure is the missing interface? And a suggestion on how to resolve this?

Reply:
In this case iperf throws an error "Cannot assign the requested ip" as iperf is trying to use the ip which is not present. The only way to resolve is to wait for the ip address to be configured before starting iperf and this should be done at line:575 before iperf starts